File: problemas/1000/1080.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace uri1080
{
    internal static class Program
    {
        private static void Main()
        {
            var inteiros = new List<int>();

            for (int i = 0; i < 100; i++)
            {
                inteiros.Add(int.Parse(Console.ReadLine()));
            }

            int maior = inteiros.Aggregate(Math.Max);
            Console.WriteLine(maior);
            Console.WriteLine(inteiros.IndexOf(maior) + 1);
        }
    }
}

File: problemas/1000/1080.dart
import 'dart:io';
import 'dart:math';

void main() {
  final inteiros = new List<int>();

  for (int i = 0; i < 100; i++) {
    inteiros.add(int.parse(stdin.readLineSync()));
  }

  int maior = inteiros.reduce(max);
  stdout.writeln(maior);
  stdout.writeln(inteiros.indexOf(maior) + 1);
}